src/styles/theming.scss

```diff
@@ -0,0 +1,24 @@
+@import '~@angular/material/theming';
```

Comment:
Why is this needed in a "Stop participation" PR?

Reply:
This file is added to provide theming for the angular material components of the applications. like checkbox